### ISSUE TITLE
Trader Duplicate Bug?

### ISSUE THREAD

**Issue by Catorii** [Dandielo/DtlCitizensTrader#68]
We are not sure how it happened at all, but we found recently that I had the blue wool block from our NPC trader - the NPC that buys things from players - in my bag. It had the caption on it still (i.e. the text that shows when you mouse over the wool block in the NPC's window), but was a stand-alone block in my bag. The only thing we could think of is somehow the block got duplicated, since the NPC can't just go missing that piece of wool, and it was still there anyway.

We tested this with the normal NPCs (the ones that sell things instead of buying) too, but we couldn't replicate the issue, so thankfully people aren't getting free stacks of items.  This is the only instance we've seen of this happening. Not sure if it's really significant or if it's just kind of a fluke thing, but we figured it's safer to report it than not to.

**Comment by Dandielo** [comment]
It's much more possible that you duplicated it in the manager mode than in the "buy/sell" mode. Because managing is quite bugged right now :/

**Comment by CyanLights** [comment]
Dandielo, I believe this also effects normal shopping. We recently discovered multiple players with multiple double chests completely full of stacks of diamond blocks. This isn't possible to get on our server. I can't say for sure it is a bug with dtl, but I sincerely think there might be a duplication glitch they are abusing with the shops.

I (or cat or k5p) will let you know if we find out more information.

**Comment by Dandielo** [comment]
I tried all possible things with shiftclick really fast window closing with items etc etc nothing has worked for me at last :D 

But If you find something, anythings, let me know

**Comment by Catorii** [comment]
It's okay, Dandielo, we found out what it was (referring to Cyan's comment). It was a bug with the NPCs (all prices reset to $0 on active traders the day that we first updated to the newer version, before you and CyanLights talked a while back) that players just took advantage of. Hopefully it won't happen again, now that it's stable, since it was the update at first that caused it. Separate issue from the manager bugs, I hope.
